pages/data-providers/[data-provider-id]/deposit-compliance.jsx

```diff
@@ -173,9 +173,12 @@ const DepositCompliance = ({
                 store.depositDates.crossDepositLag.possibleBonusCount
               ),
             })
-          : 'Loading data'}
+          : 'Data is being processed. This may take several minutes.'}
```

Comment:
How do you dustinguish 'request is being processed' and 'we wait for the response'? API does not even send you 202 in such a situation. And if it does, we should do pooling or interval requests.

It's a not good path to go.

Reply:
'request is being processed' and 'we wait for the response' is the same and I don't see a reason need to distinguish them. It doesn't matter from user point of view and it makes it more complicated...

Reply:
'Data is being processed' means you as a user send something and the system took it into account but it takes longer to process. When you did not send anything explicitly, did not press any button, it must not sound like that.

Reply:
ok so it could have been changed to something like data is being fetched/calculated.